src/tye/InitHost.cs

```diff
@@ -71,6 +72,8 @@ public static (string, string) CreateTyeFileContent(FileInfo? path, bool force)
                     // If the input file is a project or solution then use that as the name
                     application.Name = Path.GetFileNameWithoutExtension(path.Name).ToLowerInvariant();
 
+                    application.Ingress = null!;
```

Comment:
Why do you need to set this to null?

Reply:
So it doesn't show up as an empty array when you call `tye init`